cmd/minimock/minimock.go

```diff
@@ -307,6 +320,17 @@ func processArgs(args []string, stdout, stderr io.Writer) (*options, error) {
 
 	interfaces := strings.Split(*input, ",")
 
+	var mockNames []string
+	if *aliases != "" {
+		mockNames = strings.Split(*aliases, ",")
+	}
+	if len(mockNames) != 0 && len(mockNames) != len(interfaces) {
```

Comment:
Can you please make sure that -i option doesn't contain any wildcards when -n is given? This will generate mock with the same name for every interface in the package. Let's say we have -i io.* -n IOMock then we'll have same IOMock name for both io.Reader and io.Writer.

Reply:
@hexdigest done :)